File: app/src/test/java/com/tronography/pixelweather/IconUrlUtilsTest.java
package com.tronography.pixelweather;

import com.tronography.pixelweather.utils.IconUrlUtils;

import org.junit.Test;

import static org.junit.Assert.assertEquals;


public class IconUrlUtilsTest {

    @Test
    public void getIconUrlTest(){
        String icon = "02d";
        String expectedResult = "http://openweathermap.org/img/w/02d.png";

        assertEquals(expectedResult, IconUrlUtils.Companion.getIconUrl(icon));
    }
}

File: app/src/test/java/com/tronography/pixelweather/IconUrlUtilsTest.kt
package com.tronography.pixelweather

import com.tronography.pixelweather.utils.IconUrlUtils

import org.junit.Test

import org.junit.Assert.assertEquals


class IconUrlUtilsTest {

    @Test
    fun getIconUrlTest() {
        val icon = "02d"
        val expectedResult = "http://openweathermap.org/img/w/02d.png"

        assertEquals(expectedResult, IconUrlUtils.getIconUrl(icon))
    }
}
